independent-projects/bootstrap/core/src/main/java/io/quarkus/bootstrap/classloading/QuarkusClassLoader.java

```diff
@@ -351,7 +354,10 @@ public URL getResource(String unsanitisedName) {
         if (name.endsWith(".class") && !endsWithTrailingSlash) {
             ClassPathElement[] providers = state.loadableResources.get(name);
             if (providers != null) {
-                return providers[0].getResource(name).getUrl();
+                ClassPathResource resource = providers[0].getResource(name);
+                if (resource != null) {
+                    return resource.getUrl();
+                }
```

Comment:
You want to continue in this case? Just to be sure as you slightly change the logic.

Same question below.

Reply:
Yes, I'm fall-backing to `parent.getResource(unsanitisedName)`, as it's also done in the `else` statement below.